### DISCUSSION TITLE
Large Scale Fire Behavior Modeling Tool

### DISCUSSION THREAD

**Discussion by nbol48** [earthlab-education/Earth-Analytics-2023-01-Intro#56] [Awesome science]
https://www.firelab.org/project/behaveplus 
BehavePlus is a data application for fire managers built by the Fire Modeling Institute at the Rocky Mountain Research Station. It can help project the fire intensity and rate of spread under certain conditions that can assist in long term fire mitigation and preparation planning. It may also have applications (not entirely sure on how effective this is in the real world) for short to immediate term operational decisions for incident command teams looking at upcoming fire weather conditions.

**Comment by ellenalamont17** [comment]
This model seems really interesting! I didn't get a chance to read it in deep detail, but I really like that they are trying to capture the full spectrum of variables impacting fires within their model. I wonder if they are using a mixture of historic and remotely sensed data to obtain some of these parameters, because spatial coverage of some of their variables would be tricky without that.
